.env.e2e

```
@@ -30,3 +30,7 @@ HMAC_KEY=flutterwave-webhook-comparison-e2e
 FLIGHTAWARE_API_KEY=flightaware-api-key
 GOOGLE_DISTANCE_MATRIX_API_KEY=google-mapa-api-key
 SENDER_NAME=Yomide
+AWS_REGION=us-east-1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the FLIGHTAWARE_API_KEY key

(UnorderedKey)